ingestion_server/ingestion_server/indexer.py

```diff
@@ -191,6 +191,28 @@ def __init__(
     # Helpers
     # =======
 
+    def get_stat(self, name_or_alias: str) -> tuple[bool, bool, Union[str, list[str]]]:
```

Comment:
Totally not crucial, but it might be good to have this return type be a `NamedTuple`. I always get a little confused with functions returning more than two fields 😅 That way we can reference the fields under their expected names too!

Reply:
This is an awesome idea. I always forget that named tuples exist and my mind goes from structureless tuples to overkill data classes.

Reply:
Implemented in 63ee1da.